src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -30,6 +30,9 @@ type Props = {
     /** Function to retrieve the image URL associated with a pill */
     getPillImageUrl?: (data: { id: string | number, [key: string]: any }) => string,
     innerRef?: React.Ref<any>,
+    inputDisabled?: boolean,
```

Comment:
we should use an is / can / has prefix for booleans
```suggestion
    isInputDisabled?: boolean,
```